docs/installing/adding_devices.md

```diff
@@ -106,13 +106,13 @@ Now that your edge device is prepared, continue on to [Installing the agent](reg
 
 Prepare your device by installing {{site.data.keyword.rhel}} 8.x or 9.x
 
-If you are running {{site.data.keyword.rhel}} 9.0, install the podman 4.x packages.
+If you are running {{site.data.keyword.rhel}} 9.0 or above, install the podman 4.x packages.
```

Comment:
Also 8.6-8.7?

Reply:
There are specific `8.6 or above` instructions in the following line:
 
https://github.com/open-horizon/open-horizon.github.io/pull/356/commits/2aaaba92ef7089f41d867c9b3fda7a4fae45b4ce#diff-1ae267f3bdb86dcbb4e6637f8ea2ccec91a57df7501770a558b5c8842ba4a461L115